packages/calendar/src/Calendar.ts

```diff
@@ -0,0 +1,449 @@
+        const monthAndYear = this._capitalFirstLetter(
+            this._formatDate(this._currentDate, {
+                month: 'long',
+                year: 'numeric',
+            })
+        );
+
+        return html`
+            <div class="spectrum-Calendar-header">
+                <!--
+                 * TODO: Attribute 'role="heading"' removed, due to error 'The "heading" role requires the attribute
+                 * "aria-level"'
+                -->
+                <div
+                    class="spectrum-Calendar-title"
+                    aria-live="assertive"
+                    aria-atomic="true"
+                >
+                    ${monthAndYear}
+                </div>
+
+                <!-- TODO: Translate "Previous" -->
+                <sp-action-button
+                    quiet
+                    size="s"
+                    aria-label="Previous"
+                    title="Previous"
+                    class="spectrum-ActionButton spectrum-Calendar-prevMonth"
+                    ?disabled=${this.disabled}
+                    @click=${this.handlePreviousMonth}
+                >
+                    <div slot="icon">
+                        <slot name="prev-icon">
+                            <sp-icon-chevron-left></sp-icon-chevron-left>
+                        </slot>
+                    </div>
+                </sp-action-button>
+
+                <!-- TODO: Translate "Next" -->
+                <sp-action-button
+                    quiet
+                    size="s"
+                    aria-label="Next"
+                    title="Next"
+                    class="spectrum-ActionButton spectrum-Calendar-nextMonth"
+                    ?disabled=${this.disabled}
+                    @click=${this.handleNextMonth}
+                >
+                    <div slot="icon">
+                        <slot name="prev-icon">
+                            <sp-icon-chevron-right></sp-icon-chevron-right>
+                        </slot>
+                    </div>
+                </sp-action-button>
+            </div>
+        `;
+    }
+
+    public renderCalendarGrid(): TemplateResult {
+        return html`
+            <!-- TODO: Implement keyboard navigation -->
+            <div
+                class="spectrum-Calendar-body"
+                role="grid"
+                tabindex=${ifDefined(!this.disabled ? '0' : undefined)}
+                aria-readonly="true"
+                aria-disabled=${this.disabled}
+            >
+                <table role="presentation" class="spectrum-Calendar-table">
+                    ${this.renderCalendarTableHead()}
+                    ${this.renderCalendarTableBody()}
+                </table>
+            </div>
+        `;
+    }
+
+    public renderCalendarTableHead(): TemplateResult {
+        return html`
+            <thead role="presentation">
+                <tr role="row">
+                    ${this._getWeekdays().map((weekday) =>
```

Comment:
can you please make sure this._getWeekdays() should not return undefined

Reply:
This method will not return `undefined` because the number of days in a week does not change, we will always have 7 days and, consequently, an array with 7 objects with the details of each of these days

Reply:
However, I will run this method elsewhere and use a property here instead of running the function directly.

Reply:
I created a private property to store this information